package.json

```diff
@@ -4,19 +4,22 @@
   "private": true,
   "dependencies": {
     "@firebase/auth-types": "^0.10.0",
-    "@material-ui/core": "^4.9.7",
+    "@material-ui/core": "^4.9.9",
     "@material-ui/icons": "^4.9.1",
-    "@reduxjs/toolkit": "^1.2.5",
+    "@material-ui/lab": "^4.0.0-alpha.48",
+    "@reduxjs/toolkit": "^1.3.4",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
     "@types/lodash": "^4.14.149",
-    "@types/react": "^16.9.25",
+    "@types/react": "^16.9.32",
     "@types/react-redux": "^7.1.7",
     "@types/react-router-dom": "^5.1.3",
     "awesome-typescript-loader": "^5.2.1",
+    "better-docs": "^1.4.7",
     "express": "^4.17.1",
-    "firebase": "^7.12.0",
+    "firebase": "^7.13.2",
+    "jsdoc": "^3.6.4",
```

Comment:
You should add this to `devDependencies`.
Probably best to leave the other packages the way they are unless there's a specific patch you need?

Reply:
Oops, my bad.  Reverted.